pages/linux/gdrive.md

```diff
@@ -0,0 +1,20 @@
+# gdrive
+
+> Command line tool to interact with Google Drive.
+> Folder/file id can be obtained from the Google Drive folder or id url.
+
+- Upload a local path to the parent folder with the specified id:
+
+`gdrive upload -p {{id}} {{path/to/file_or_folder}}`
+
+- Download file or directory by id to current directory:
+
+`gdrive download {{id}}`
+
+- Download to a given local path by its id to a given local path:
+
+`gdrive download --path {{path/to/file_or_folder}} {{id}}`
+
+- Create a new revision of an id using a given file:
```

Comment:
"given file or folder" - to match with the example

Reply:
Sorry, I would have done this change myself, but you don't seem to have given me edit permissions. Kindly take a look at this. :)